mmeds/server.py

```diff
@@ -716,11 +733,7 @@ def continue_metadata_upload(self):
         # Move on to uploading data files
         if cp.session['metadata_type'] == 'specimen':
```

Comment:
The logic of this function `continue_metadata_upload` is a bit unclear to me. Why are we checking that `specimen` is the metadata_type, and what does it mean? What metadata are there available? From the line below it seems if this value is *not* `specimen`, then it gets overwritten?

Reply:
Ok, that makes sense. Put somewhere in comments there, we need to capture this type of knowledge in documentation.